pkg/cmd/template/render/parameters.go

```diff
@@ -0,0 +1,156 @@
+package render
+
+import (
+	"fmt"
+	"github.com/coreeng/corectl/pkg/cmdutil/userio"
+	"github.com/coreeng/corectl/pkg/template"
+	"gopkg.in/yaml.v3"
+	"os"
+	"slices"
+	"strings"
+)
+
+func CollectArgsFromAllSources(
+	spec *template.Spec,
+	argsFile string,
+	flagArgsRaw []string,
+	streams userio.IOStreams,
+	existingArgs []template.Argument,
+) ([]template.Argument, error) {
+	fileArgs, err := parseArgsFile(spec, argsFile)
+	if err != nil {
+		return nil, err
+	}
+
+	flagArgs, err := parseArgsFromFlags(spec, flagArgsRaw)
```

Comment:
if I have an argument called `tenant` and I execute corectl `app create --tenant tenant-name` will it work or do I need to do `app create --tenant tenant-name -a tanant=tenant-name` ?

Reply:
it will propagate `name` and `tenant` args automatically.
But only for the `app create` command